.github/workflows/pull-request.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [18.x, 20.x]
 
     steps:
     - uses: actions/checkout@v2
```

Comment:
I believe you should update this file as part of your Karma test update. However, please make sure to update all the references to `v2` actions to `v3` as this version includes a more current node version.